haskell/src/Indigo/Contracts/Transferlist/External.hs

```diff
@@ -0,0 +1,77 @@
+-- SPDX-FileCopyrightText: 2020 Globacap
```

Comment:
It's actually a bit weird. Did you write `Globacap` here because you think you copy-paste a substantial copy of https://github.com/serokell/tezos-globacap/blob/master/nettest/Nettest/WhitelistIntegration.hs? I think only `Storage` definitions sort of resembles `WhitelistStorage`, the rest looks like a new code.

Reply:
done